### DISCUSSION TITLE
`zsh: segmentation fault `

### DISCUSSION THREAD

**Discussion by yunkai-jia** [dashingsoft/pyarmor#1166] [Q&A] [answered]
(base) users@users Desktop % pyarmor reg pyarmor-regfile-5013.zip 
INFO     Python 3.9.7
INFO     Pyarmor 8.1.0 (trial), 000000, non-profits
INFO     register "pyarmor-regfile-5013.zip"
INFO     update license token
zsh: segmentation fault  pyarmor reg pyarmor-regfile-5013.zip
(base) users@users Desktop %

**Comment by jondy** [comment, marked as answer]
duplicated of #1155

**Comment by yunkai-jia** [comment]
谢谢～
